src/ape_geth/query.py

```diff
@@ -1,12 +1,22 @@
 from functools import singledispatchmethod
 from typing import Iterator, Optional
 
+from pydantic import BaseModel
+
 from ape.api import ReceiptAPI
 from ape.api.query import ContractCreationQuery, QueryAPI, QueryType
 from ape.exceptions import QueryEngineError
+from ape.types.address import AddressType
 from ape_ethereum.provider import EthereumNodeProvider
 
 
+class ContractCreation(BaseModel):
```

Comment:
Should this be defined here? This seems like something that should live in the API level, since other plugins like `ape-etherscan` also implement this.

Reply:
resolved in https://github.com/ApeWorX/ape/pull/2001/commits/65446305c255d94c7a416ec8913262b08f5d8f48